<launch>
    <node name="cam_capture_node" pkg="vapaa_cruiser" type="cam_capture_node.py" output="screen">
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <param name="rate" value="30"/>
        <param name="flipMode" value="0"/>
    </node>
    <node name="cam_calibration_node" pkg="vapaa_cruiser" type="cam_calibration_node.py" output="screen" cwd="node">
        <param name="gridX" value="8"/>
        <param name="gridY" value="6"/>
        <param name="gridSize" value="25"/>
        <param name="targetW" value="320"/>
        <param name="targetH" value="240"/>
    </node>
    
</launch>
